flink-table/flink-table-planner/src/test/resources/META-INF/services/org.apache.flink.table.factories.Factory

```diff
@@ -13,8 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-org.apache.flink.formats.testcsv.TestCsvFormatFactory
```

Comment:
Why change order?

Reply:
> Why change order?

When removing `TestManagedTableFactory` from the file, I change the order by mistake, will sort them alphabetically